pkg/kubelet/kubelet_test.go

```diff
@@ -1842,6 +1842,51 @@ func TestHandlePodAdditionsInvokesPodAdmitHandlers(t *testing.T) {
 	checkPodStatus(t, kl, podToAdmit, v1.PodPending)
 }
 
+// Test verifies that the kubelet invokes an admission handler during HandlePodAdditions when nodeinfo not ready.
+func TestHandlePodAdditionsInvokesPodAdmitHandlersWithNodeinfoNotReady(t *testing.T) {
```

Comment:
Nit: Test name should be `...WithNodeInfoNotReady` to properly follow CamelCase.